accelerators/federated-learning/automated_provisioning/ReadMe.md

```diff
@@ -0,0 +1,107 @@
+
+## Contents
+
+This document proposes a procedure to automatically provision the resources required for running Contoso Federated Learning experiments.
+
+## Requirements
+
+For running Federated Learning experiments, the Contoso team needs the following ingredients:
+
+- an "orchestrator" Azure ML workspace;
+- some Kubernetes (K8s) clusters;
+- connections between the K8s clusters and the Azure ML workspace;
+- <other ingredients related to data store, to be added later>...
+
+The procedure outlined in this document will provision resources that meet the requirements above.
+
+## Prerequisites
+
+Taken from [here](https://github.com/Azure/AML-Kubernetes#prerequisites) (along with the K8s cluster creation/connection steps).
+
+1. Have access to an Azure subscription.
+2. Install the [latest release of Helm 3](https://helm.sh/docs/intro/install/) - for Windows, we recommend going the _chocolatey_ route.
+3. Meet the pre-requisites listed under the [generic cluster extensions documentation](https://docs.microsoft.com/azure/azure-arc/kubernetes/extensions#prerequisites).
+    - Azure CLI version >=2.24.0
+    - Azure CLI extension k8s-extension version >=1.0.0.
+4. Install and setup the [latest AzureML CLI v2](https://docs.microsoft.com/azure/machine-learning/how-to-configure-cli).
+5. Install the [Bicep CLI](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/install) and the [Bicep extension in VS Code](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep).
+
+## Procedure
+
+> Set `accelerators/federated_learning/automated_provisioning` as your working directory!
+
+The procedure is fairly simple (detailed instructions are given in the below sections).
+1. After setting your working directory and  verifying you are meeting the prerequisites, you will first run the `CreateK8sCluster.ps1` script (with the appropriate arguments) and log in when prompted.
+2. Then you will run the `ConnectSiloToOrchestrator.ps1` script (with the appropriate arguments) and log in when prompted.
+
+We also suggest a way to run a simple validation job (highly recommended, to make sure the newly provisioned resources are usable). That being said, there is nothing special about this job, except for the fact that it will run on the new Arc-enabled K8s cluster; if you feel more comfortable using one of your own jobs to achieve that, it is perfectly acceptable.
+
+### Set up a silo
+
+For starters, you need to create a K8s cluster and the associated resource group if they don't exist already. Then you'll need to connect to this cluster and create the _kube_ config file (which will be used implicitly by the second script to point at this particular cluster). There is a script that does all that for you: `CreateK8sCluster.ps1`. It takes the following input arguments.
+- `SubscriptionId`: the Id of the subscription where the silo will be created (default: "48bbc269-ce89-4f6f-9a12-c6f91fcb772d", _a.k.a._ the AIMS subscription). 
```

Comment:
Suggestion: remove the default, since this subscription ID won't mean anything outside our team.